tests/rbd/rbd_utils.py

```diff
@@ -730,3 +763,41 @@ def rbd_remove_image_negative_validate(rbd, pool, image):
         log.error(f"{out}")
         raise ImageIsDeletedError(f" RBD image is deleted: {out}")
     log.info(f"{err}")
+
+
+def verify_migration(rbd, dest_spec):
+    """
+    This method will verify for the Live migration is prepared or not
+    Args:
+        rbd: rbd object
+        dest_spec: Target image spec in format of TARGET_POOL_NAME/SOURCE_IMAGE_NAME
```

Comment:
Can we please add information about this method being able to verify both migration preparation and migration execution ?

Reply:
Would be great if method name can be modified.
Method name is misleading, actually its verifying sub-steps of migration,not actual migration

Reply:
Done.